web/firefoxcom.js

```diff
@@ -209,6 +209,10 @@ PDFViewerApplication.externalServices = {
     FirefoxCom.request('updateFindControlState', data);
   },
 
+  updateFindMatchesCount(data) {
+    // FirefoxCom.request('updateFindMatchesCount', data);
```

Comment:
Note that this is purposely commented out, for now, since https://searchfox.org/mozilla-central/rev/37663bb87004167184de6f2afa6b05875eb0528e/browser/extensions/pdfjs/content/PdfStreamConverter.jsm#719,740 will throw without the `mozilla-central` follow-up patch.